source/boards/navigate-boards.rst

```diff
@@ -0,0 +1,64 @@
+Navigate boards
+===============
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+  
+If you're using Mattermost 7.0, open boards using the App Bar. If you don't see the Apps Bar to the right of your screen, you can access boards using the product menu in the top left corner of Mattermost.
```

Comment:
Recommend updating or removing this section. If a channel doesn't have linked boards, then you can't open Boards directly from the Apps Bar.